app/javascript/components/fonticon-picker/font-icon-picker-ddf.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import {
+  ButtonGroup,
+  Icon,
+} from 'patternfly-react';
```

Comment:
please remove this, i think we are not using this anymore.

Reply:
ok